diff --git a/modules/social_features/social_comment/social_comment.module b/modules/social_features/social_comment/social_comment.module
--- a/modules/social_features/social_comment/social_comment.module
+++ b/modules/social_features/social_comment/social_comment.module
@@ -69,12 +69,9 @@
       break;

     default:
-      // Redirect to the current url. We should not use this change for posts.
-      // See social_post_form_comment_post_comment_form_alter for posts.
-      if ($form_id !== 'comment_post_comment_form') {
-        $form['#action'] = \Drupal::request()->getRequestUri();
-        $form['field_comment_body']['widget'][0]['#placeholder'] = t('Add a comment');
-      }
+      // Remove check for type to always post comments without confirmation
+      $form['#action'] = \Drupal::request()->getRequestUri();
+      $form['field_comment_body']['widget'][0]['#placeholder'] = t('Add a comment');
       break;
   }

